commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/service/FileListSerializer.kt

```diff
@@ -6,17 +6,25 @@ import ac.uk.ebi.biostd.integration.SubFormat
 import ac.uk.ebi.biostd.validation.InvalidChunkSizeException
 import ebi.ac.uk.errors.FileNotFoundException
 import ebi.ac.uk.io.sources.FilesSource
+import ebi.ac.uk.model.BioFile
 import ebi.ac.uk.model.FileList
-import ebi.ac.uk.model.FilesTable
 import ebi.ac.uk.model.Submission
 import ebi.ac.uk.model.extensions.allSections
 import ebi.ac.uk.model.extensions.fileListName
+import ebi.ac.uk.util.file.ExcelReader.asTsv
+import mu.KotlinLogging
 import java.io.File
+import java.io.InputStream
+
+private val logger = KotlinLogging.logger {}
 
 internal class FileListSerializer(
-    private val serializer: PagetabSerializer
+    private val serializer: PagetabSerializer,
```

Comment:
is the standart practice since kotlin [1.4](https://kotlinlang.org/docs/whatsnew14.html#trailing-comma) so trying to adopting. Basically it helps when new parameter is added